pages/linux/xdotool.md

```diff
@@ -9,3 +9,19 @@
 - Click the right mouse button:
 
 `xdotool click {{3}}`
+
+- Get the id of the currently active window:
+
+`xdotool getactivewindow`
+
+- Focus on window with the id of 12345:
+
+`xdotool windowfocus --sync 12345`
+
+- Send a Hello World message with 500ms delay for each letter:
```

Comment:
Not sure that `Hello World` needs to be in title case here.

Also, perhaps `message, with a 500ms` would read better?